.github/workflows/run-tests.yml

```diff
@@ -47,7 +61,21 @@ jobs:
         with:
           path: neofs-node
 
-################################################################
+      - name: Download latest stable neofs-cli for uploading reports to NeoFS
+        uses: dsaltares/fetch-gh-release-asset@1.1.1
+        with:
+          repo: 'nspcc-dev/neofs-node'
+          version: 'tags/v0.37.0'
+          file: 'neofs-cli-amd64'
+          target: 'neofs-node-stable/neofs-cli'
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It's an open repo, I don't think we need it.

Reply:
> It's an open repo, I don't think we need it.

Fixed